pallets/providers/src/utils.rs

```diff
@@ -992,7 +1195,9 @@ where
     /// being an exact match to a file key stored by the Storage Provider. The StorageHub protocol requires the Storage Provider to
     /// submit a proof of storage for the neighbouring file keys of the missing challenged file key.
     ///
-    /// The slashing amount is calculated based on an assumption that every file is the maximum size allowed by the protocol.
+    /// The slashing amount is calculated as the product of the [`SlashAmountPerMaxFileSize`](Config::SlashAmountPerMaxFileSize) (an assumption
+    /// that every file is the maximum size allowed by the protocol) and the accrued failed proof submissions multiplied by `2` to
```

Comment:
I don't think we enforce a max file size. This just a "very large file size" that we consider. Meaning that most files should be below that.